src/core/src/core_logic/PatchInstaller.py

```diff
@@ -94,8 +94,20 @@ def start_installation(self, simulate=False):
         overall_patch_installation_successful = bool(update_run_successful and not maintenance_window_exceeded)
         if overall_patch_installation_successful:
             self.status_handler.set_installation_substatus_json(status=Constants.STATUS_SUCCESS)
+            # update patch metadata in status for auto patching request, to be reported to health store
+            if self.execution_config.patch_rollout_id is not None:
+                patch_version = None
+                try:
+                    patch_version = self.env_layer.datetime.utc_to_standard_datetime(self.execution_config.patch_rollout_id).date()
+                except ValueError as e:
+                    patch_version = Constants.PATCH_VERSION_UNKNOWN
```

Comment:
The execution cannot continue with this assumption. This will just result in perpetual retries until the problem goes away. If this happens, we need to log, stop and fail (until such time that rollout id changes to more than just a datetime).

In the future, this will be a pass through but there's no legitimate reason this should happen, and no reason to continue from here.

Reply:
Bascially, log error and throw to stop execution.